ERCS/erc-7871.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 7871
+title: Wallet Signing API
+description: Adds a JSON-RPC method for requesting a signature from a wallet
+author: Lukas Rosario (@lukasrosario), Jake Moxey (@jxom), Cody Crozier (@wcrozier12), Conner Swenberg (@ilikesymmetry)
+discussions-to: https://ethereum-magicians.org/t/new-erc-wallet-signing-api/22718
+status: Draft
+type: Standards Track
+category: Interface
+created: 2024-01-29
+requires: 191, 712, 5792
+---
+
+## Abstract
+
+Defines a new JSON-RPC method which enables apps to ask a wallet to sign [EIP-191](./eip-191.md) messages.
+
+Applications can use this JSON-RPC method to request a signature over any version of `signed_data` as defined by [EIP-191](./eip-191.md). The new JSON-RPC method allows for support of future [EIP-191](./eip-191.md) `signed_data` versions.
+
+The new JSON-RPC method also supports [EIP-5792](./eip-5792.md)-style `capabilities`, and support for signing capabilities can be discovered using `wallet_getCapabilities` as defined in [EIP-5792](./eip-5792.md).
```

Comment:
You abstract does a good job at conveying the "what" of this proposal, but is somewhat lacking on the "how" / "why". There should be enough depth here to give the reader a high-level (but still technical) overview of the proposal.